qiskit/optimization/problems/quadratic_program.py

```diff
@@ -53,6 +54,16 @@ class QuadraticProgramStatus(Enum):
     INFEASIBLE = 1
 
 
+def _update_var_list(var_list: List[Variable], variable: Variable) -> List[Variable]:
```

Comment:
You don't use `_update_var_list` and `_update_var_dict`. Could you clean them up?